docusaurus.config.js

```diff
@@ -0,0 +1,150 @@
+
+  // Even if you don't use internationalization, you can use this field to set
+  // useful metadata like html lang. For example, if your site is Chinese, you
+  // may want to replace "en" with "zh-Hans".
+  i18n: {
+    defaultLocale: 'en',
+    locales: ['en'],
+  },
+
+  presets: [
+    [
+      'classic',
+      /** @type {import('@docusaurus/preset-classic').Options} */
+      ({
+        docs: {
+          routeBasePath: '2.x/docs/',
+          sidebarPath: './sidebars.js',
+          // Please change this to your repo.
+          // Remove this to remove the "edit this page" links.
+          editUrl:
+            'https://github.com/sbt/website/tree/develop',
+        },
+        theme: {
+          customCss: './src/css/custom.css',
+        },
+      }),
+    ],
+  ],
+
+  themeConfig:
+    /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
+    ({
+      // Replace with your project's social card
+      image: 'img/docusaurus-social-card.jpg',
+      navbar: {
+        // title: 'sbt',
+        logo: {
+          alt: 'sbt logo',
+          src: 'assets/sbt-logo.svg',
+        },
+        items: [
+          {
+            href: '/learn',
+            label: 'Learn',
+            position: 'left',
+          },
+          {
+            href: '/download',
+            label: 'Download',
+            position: 'left',
+          },
+          {
+            href: '/community',
+            label: 'Get Involved',
+            position: 'left',
+          },
+          {
+            type: 'docSidebar',
+            sidebarId: 'tutorialSidebar',
+            label: 'Doc Beta',
+            position: 'right',
+          },
+          {
+            href: 'https://github.com/sbt/website',
+            label: 'GitHub',
+            position: 'right',
+          },
+        ],
+      },
+      footer: {
+        style: 'dark',
+        links: [
+          {
+            title: 'Docs',
+            items: [
+              {
+                label: 'sbt 1.x docs',
+                to: 'https://www.scala-sbt.org/1.x/docs/',
+              },
+            ],
+          },
+          {
+            title: 'Community',
+            items: [
+              {
+                label: 'Discord',
+                href: 'https://discord.com/channels/632150470000902164/922600050989875282',
+              },
+              {
+                label: 'Stack Overflow',
+                href: 'https://stackoverflow.com/questions/tagged/sbt',
+              },
+            ],
+          },
+          {
+            title: 'More',
+            items: [
+              {
+                label: 'GitHub',
+                href: 'https://github.com/sbt/sbt',
+              },
+            ],
+          },
+        ],
+        copyright: `Copyright © ${new Date().getFullYear()} Scala Center. Built with Docusaurus.`,
+      },
+      prism: {
+        theme: prismThemes.github,
+        darkTheme: prismThemes.dracula,
+      },
+      algolia: {
```

Comment:
So no idea if you've had a good experience with algolia or not, but in the other repos I've maintained we had weird issues with it. One really great alternative that I found and that we use in Metals and the BSP repos is https://github.com/scalameta/metals/blob/167d069ddd7d6f7965e67fa9cfe39b12cd1977bb/website/docusaurus.config.js#L56 so you still get site search, but it's all generated locally. So there is no need for the external algolia integration. Just thought I'd drop this here if you're interested at all.

Reply:
We already use Algolia, and as far as I can tell, it works pretty well https://www.scala-sbt.org/. I'd be happy to iterate on this if local search is in fact better?

Reply:
Nah, I don't know if it's _better_. Just if you ever hit on issues, know that there is an algolia alternative.